skopt/tests/test_acquisition.py

```diff
@@ -17,9 +20,27 @@
 
 class ConstSurrogate:
     def predict(self, X, return_std=True):
+        X = np.array(X)
         return np.zeros(X.shape[0]), np.ones(X.shape[0])
 
 
+class MultiOutputSurrogate:
```

Comment:
Its more like a two output surrogate, we might want to add multi - output surrogates later, might be better to spare this name for now to use it later. Could you use something along the lines of "TwoOutputSurrogate"?